virtcontainers/hypervisor.go

```diff
@@ -399,6 +399,9 @@ type HypervisorConfig struct {
 	// VMid is the id of the VM that create the hypervisor if the VM is created by the factory.
 	// VMid is "" if the hypervisor is not created by the factory.
 	VMid string
+
+	// SELinux label for the VM
+	ProcessLabel string
```

Comment:
Currently only one LSM is supported at a time, and AppArmor is not label based.

We originally called this ProcessLabel, so that a different labeling LSM like Smack could be used and would use the same infrastructure.  Over time this has not proven to be important, and slowly SELinux has snuck it's way into the tool.  This is why the commands are label rather then just calling it SELinux directly.